pkg/kubelet/kubelet.go

```diff
@@ -3286,6 +3286,14 @@ func (kl *Kubelet) generateAPIPodStatus(pod *api.Pod, podStatus *kubecontainer.P
 	s.Phase = GetPhase(spec, s.ContainerStatuses)
 	kl.probeManager.UpdatePodStatus(pod.UID, s)
 	s.Conditions = append(s.Conditions, status.GeneratePodReadyCondition(spec, s.ContainerStatuses, s.Phase))
+	// We want to keep PodScheduled condition. If for any reason it's not set we should do it now as the pod
```

Comment:
@yujuhong For some reason this block breaks the node. I don't know why but it breaks docker daemon in a very reproducible way:
```$ sudo service docker status [FAIL] docker is not running ... failed!```
When I comment these 5 lines out it works.

Can you help me understand this?